curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dfa22d1b521be39a3de7be0.md

```diff
@@ -21,22 +21,23 @@ In the text of your `p` element, turn the words `cat photos` into a link by addi
 You should nest a new anchor (`a`) element within the `p` element.
 
 ```js
-assert($('p > a').length);
+const nestedAnchor = document.querySelector(`p > a`);
+assert(nestedAnchor);
 ```
 
 The link's `href` value should be `https://freecatphotoapp.com`. You have either omitted the `href` value or have a typo.
 
 ```js
-const nestedAnchor = $('p > a')[0];
+const nestedAnchor = document.querySelector(`p > a`);
 assert(
-  nestedAnchor.getAttribute('href') === 'https://freecatphotoapp.com'
+  nestedAnchor.getAttribute('href') === 'https://freecatphotoapp.com' || nestedAnchor.getAttribute('href') === 'https://freecatphotoapp.com/'
```

Comment:
I noticed when I looked at this that the value returned from the `href` property on the element always has a trailing slash no matter if it was written in the code with or without it. When using `getAttribute` it changes.

So the test should be able to just check for that attribute value. I only tested this with Chrome and Firefox but I believe looking at [the specs](https://url.spec.whatwg.org/#concept-url-serializer) for it, it should be the same in all browsers.

```
document.querySelector('p > a').href === 'https://freecatphotoapp.com/'
```

That might be a bit cleaner. But both work.

Reply:
updated